index.html

```diff
@@ -1868,6 +1880,32 @@ <h2>Contexts and Vocabularies</h2>
 `curl -sL -H "Accept: &lt;MEDIA_TYPE>" &lt;DOCUMENT_URL> | openssl dgst -&ltDIGEST_ALGORITHM> -binary | openssl base64 -nopad -a`.
         </p>
 
+        <p>
+Application-specific vocabulary and specification authors SHOULD ensure that
+their JSON-LD Context and vocabulary files are permanently cacheable using
+either the approaches to caching as described above, or any other functionally
+equivalent mechanism.
+        </p>
+
+        <p>
+Implementations MAY load application-specific JSON-LD Context files from the
+network during development, but SHOULD permanently cache JSON-LD Context files
+utilized in <a>conforming documents</a> in production settings to increase the
+security and privacy characteristics of solutions. Approaches to caching as
+described above, or functionally equivalent mechanisms, MAY be utilized to
+achieve caching goals.
+        </p>
+        <p>
+Some applications, such as digital wallets that are capable of holding arbitrary
+<a>verifiable credentials</a> (from any issuer, using any contexts) or other
+data-integrity-protected documents, might need to be able to load externally
+linked resources, such as JSON-LD Context files, in production settings. This is
+expected to increase user choice, scalability, and decentralized upgrades in the
+ecosystem over time. Authors of these applications are advised to read the
+security and privacy sections of this document for further considerations.
+        </p>
+        </p>
+
         <p>
 For further information regarding processing of JSON-LD Contexts and
 Vocabularies, see <a data-cite="?VC-DATA-MODEL-2.0#base-context">Verifiable
```

Comment:
Taking my `contexts` and `vocabularies` lowercasing cue from https://www.w3.org/TR/json-ld11/ where they are only capitalized in section titles.
```suggestion
For further information regarding processing of JSON-LD contexts and
vocabularies, see <a data-cite="?VC-DATA-MODEL-2.0#base-context">Verifiable
```